content/blog/2022-03-31-training-and-saving-models-on-a-dedicated-runner.md

```diff
@@ -0,0 +1,377 @@
+---
+title: Training and saving models with CML on a dedicated runner
+date: 2022-03-31
+description: Hello World
+descriptionLong: |
+  Some long
+  multiline
+  text (supports _basic_ [Markdown](link))
+picture: 2015-05-01/post-image.jpeg
+pictureComment: Some _Comment_ (supports _basic_ [Markdown](link))
+author: rob_dewit
+# commentsUrl: https://discuss.dvc.org/t/february-22-community-gems/1078
+tags:
+  - CML
+  - DVC
+  - DVC Remotes
+  - Git
+  - Pipelines
+---
+
+# Training and saving models with CML on a dedicated runner
+
+When you first develop a machine learning model, you will probably do so on your
+local machine. You can easily change algorithms, parameters, and input data
+right in your text editor, notebook, or terminal. But what happens when you
+already have a model deployed and want to run multiple experiments? Or if you
+want to deploy a new version daily?
+
+[CML (Continuous Machine Learning)](https://cml.dev/) helps you with this. It is
+an open-source library for implementing continuous integration and delivery
+(CI/CD) in machine learning projects. This way we can define a pipeline to train
+a model and keep track of various versions.
+
+> Imagine we work for the fraud detection team of a webshop and we want to build
+> a model to detect fraudulent orders. One week fraudsters might be trying to
+> order gaming consoles, while the next they might go all-in on diapers. In that
+> case, we would probably want to retrain our model regularly to catch this
+> [drift](https://en.wikipedia.org/wiki/Concept_drift). And we would also want
+> to use the most up-to-date version of our model in production.
+
+In this guide, we will show how you can use CML to (re)train your model daily
+and save its results. Although we could train the model directly in our CI/CD
```

Comment:
This is probably a better intro. Consider rewriting the title along these lines and consolidating the multiple paragraphs above this as they're somewhat confusing and/or repetitive.

Reply:
I second this comment. This would be a good part of the intro. Maybe move it to be the second paragraph?